spec/system/user/view_user_index_spec.rb

```diff
@@ -10,6 +11,8 @@
   context "when user is unauthorized" do
```

Comment:
This spec fails bc its in the "the user is unauthorized" context. If you move it down to the "when visiting own profile" context then it passes.

Reply:
Yes, this is fixed locally, I just never pushed it because the PR went back to draft and I'm not sure if I'll continue working on it the current form or if a more comprehensive profile page revamp will be done right away.